src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1522,8 +1533,13 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string,
   }
 
   if (PTUIterationType::RetryIteration == iteration_type) {
-    MessageHelper::SendPolicySnapshotNotification(
-        GetAppIdForSending(), pt_string, std::string(""), application_manager_);
+    uint32_t app_id_for_sending = GetAppIdForSending();
+
+    if (0 != app_id_for_sending) {
+      MessageHelper::SendPolicySnapshotNotification(
+          app_id_for_sending, pt_string, std::string(""), application_manager_);
```

Comment:
@IGapchuk maybe just `std::string()` ?

Reply:
@AKalinich-Luxoft Done in 0b50b09dccb77273e6047d277a99ef2ad28baaac